client/signup/config/steps-pure.js

```diff
@@ -383,9 +383,40 @@ export function generateSteps( {
 			providesDependencies: [ 'siteTopic' ],
 		},
 
-		'site-information': {
```

Comment:
Can we keep a version of the `site-information` step as it currently is for flows without the preview (currently `onboarding`)?  We can maybe change the `informationType` prop to an array, so that it can support more than one field in a step.

Reply:
> We can maybe change the informationType prop to an array, so that it can support more than one field in a step.

It's just so crazy it might work, captain! 🖖 I'll refactor.